src/init.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+
+import argparse
+import os
+
+from kubernetes import client, config
+from kubernetes.client import ApiClient, Configuration
+
+from openshift.dynamic import DynamicClient
+
+parser = argparse.ArgumentParser(description="Options to Program")
+parser.add_argument('-n',default="openshift-machine-api",dest="namespace",help="Namespace to use")
+parser.add_argument('-c',default="/config/env/CLUSTERID",required=False,dest="clusteridfile",help="Location to write CLUSTERID")
+parser.add_argument('-r',default="/secrets/aws/config.ini",required=False,dest="regionfile",help="Location to write AWS region config.ini")
+parser.add_argument('-o',default="/secrets/aws/credentials.ini",required=False,dest="credsfile",help="Location to write AWS credentials.ini")
+parser.add_argument('-a',default="/secrets/aws_access_key_id",required=False,dest="accesskeyinfile",help="Source of AWS access key")
+parser.add_argument('-A',default="/secrets/aws_secret_access_key",required=False,dest="secretkeyinfile",help="Source of AWS secret key")
+
+args = vars(parser.parse_args())
+
+def ensure_dir(filepath):
+  if not os.path.exists(os.path.dirname(filepath)):
```

Comment:
Spacing should also be 4 spaces not 2

Reply:
Oh dear. That's going to be more annoying to fix.

Reply:
`sed -i 's/^\ \ /\ \ \ \ /' init.py` should do the trick. I think you need `sed -i ''` on OSX though :)